examples/README.md

```diff
@@ -531,4 +520,130 @@ Yay !  You did it !
+	result, err = contract.EvaluateTransaction("retrieveAsset", "asset1")
+	if err != nil {
+		logger.Fatalf("Failed to Evaluate transaction: %v", err)
+	}
+	logger.Infof("--> Result: %s", string(result))
+}
+```
+
+As you can see in the go imports, we are going to use the Fabric SDK Gateway package together with the FPC Client SDK Gateway package (which we name `fpc` here).
+
+The main function consists just of a few lines of code.
+To focus on the use of the FPC Client SDK we omit to go thourgh the entire process of setting up the Fabric Gateway.
+Instead, we create a `network` instance of the type [gateway.Network](https://pkg.go.dev/github.com/hyperledger/fabric-sdk-go/pkg/gateway#Network) by using `utils.SetupNetwork("mychannel")` which implements the normal gateway setup process.
+The `network` instance represets a Fabric Network with the channel specified in `channelID`.
+
+Similar to the Fabric SDK, we create a `contract` instance by using `fpc.GetContract(network, ccID)`. 
+This function receives a `network` instance and the chaincode ID `ccID`.
+As you can see in the code, we read the chaincode ID and the channel ID from the environment variables `CC_ID` and `CHAN_ID`.
+
+The `contract` represents the FPC Chaincode which we interact with using the `SubmitTransaction` and the `EvaluateTransaction` methods.
+If you are not familiar with the gateway concept, `SubmitTransaction` corresponds to a `peer chaincode invoke` call and `EvaluateTransaction` corresponds to a `peer chaincode query` call.
+Both methods require one or more arugments as strings.
+The first argument is the chaincode function to invoke. In this tutorial, the second argument is the name of the asset followed by the value as third argument.
+
+In the example code we submit a transaction (`SubmitTransaction`) to store a new asset `asset1` with the value `100` by invoking the `storeAsset` function of the chaincode.
+To retrieve the value of the asset `asset1` stored on the ledger we invoke the `retrieveAsset` function of the chaincode using `EvaluateTransaction`.
+
+Also note that the transaction arguments and the response are encrypted while in transit.
+That is, the Fabric Client SDK encryptes the transaction arguments using the Chaincode Encryption Key associated with the Chaincode at the FPC Enclave Registry.
+The transaction arguments are then decrypted inside the FPC enclave.
+The transaction arguments also contain a Response Encryption Key, which is generated by the Fabric Client SDK.
+This key is then used by the FPC enclave to encrypt the response. When the Fabric Client SDK receives the response, it decrypts the response and returns in clear.
+
+### Run the app
+
+To run our client application we will use the `test.sh` we used in the previous section.
+Just add the following lines at the end of the `run_test()` function in `test.sh`:
+```bash
+say "- interact with the FPC Chaincode using our client app"
+export CC_ID
+export CHAN_ID
+go run client_app/
```

Comment:
go run client_app/ doesn't work
it should be:
    CLIENTAPP="./client_app"
    go run $CLIENTAPP
    still after changing that I still got the following error:
test.sh: - interact with the FPC Chaincode using our client app
# github.com/hyperledger-labs/fabric-private-chaincode/examples/helloworld/client_app
client_app/helloworld.go:8:2: imported and not used: "github.com/hyperledger/fabric-sdk-go/pkg/gateway"

Reply:
>go run client_app/ doesn't work
it should be:
CLIENTAPP="./client_app"
go run $CLIENTAPP

Oh, good catch. Let's use then the shorter version `go run client_app/helloworld.go`

Reply:
> client_app/helloworld.go:8:2: imported and not used: "github.com/hyperledger/fabric-sdk-go/pkg/gateway"

Fixed